From ba9a88b8852667b055edd70c836bb8e43ab11b8b Mon Sep 17 00:00:00 2001
From: Sergey Sharybin <sergey@blender.org>
Date: Thu, 27 Mar 2025 17:59:16 +0100
Subject: [PATCH] Skip installation of baked hipfb files

The files are included into the library itself as data, there is no
need to install them, as they are not supposed to be used.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b13d2da..790ce6c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -585,7 +585,7 @@ install(FILES ${HIPRT_ORO_HEADERS}
 		DESTINATION include/contrib/Orochi/ParallelPrimitives)
 
 # add hipfb files
-if(PRECOMPILE)
+if(PRECOMPILE AND NOT BAKE_COMPILED_KERNEL)
 	install(FILES ${KERNEL_HIPRT_COMP} ${KERNEL_OROCHI_COMP}
 			DESTINATION bin)
 endif()
-- 
2.48.1

